docs/guides/rate-limits-project-based.mdx

```diff
@@ -0,0 +1,100 @@
+
+## Types of Rate Limits
+
+Ory Network implements two types of rate limits for each policy:
+
+1. **Burst Limit:** Limits the number of requests per second, allowing brief spikes in request volume.
+2. **Sustained Limit:** Limits the number of requests over a one-minute period, ensuring consistent performance over time.
+
+## Rate Limit Policies
+
+Ory applies different rate limit policies based on the path of your request:
+
+1. `/sessions/whoami`: Session checks
+2. `/admin/oauth2/introspect`: OAuth2 token introspection
+3. `/relation-tuples/check`: Permission checks
+4. `GET /admin/identities`: List identities
+5. `*`: All other paths
+
+## Rate Limit Tables by Plan
+
+### Developer Plan Rate Limits
+
+| Environment   | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------ | :------------------------- | ----------: | --------------: |
+| **Developer** | `/sessions/whoami`         |          10 |             300 |
+|               | `/admin/oauth2/introspect` |          10 |             300 |
+|               | `/relation-tuples/check`   |          10 |             300 |
+|               | `GET /admin/identities`    |           1 |              10 |
+|               | `*`                        |           5 |             150 |
+
+### Production Plan Rate Limits
+
+| Environment    | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------- | :------------------------- | ----------: | --------------: |
+| **Production** | `/sessions/whoami`         |          80 |            1800 |
+|                | `/admin/oauth2/introspect` |          80 |            1800 |
+|                | `/relation-tuples/check`   |          80 |            1800 |
+|                | `GET /admin/identities`    |          10 |             300 |
+|                | `*`                        |          40 |             900 |
+| **Staging**    | `/sessions/whoami`         |          10 |              30 |
+|                | `/admin/oauth2/introspect` |          10 |             300 |
+|                | `/relation-tuples/check`   |          10 |             300 |
+|                | `GET /admin/identities`    |           1 |              10 |
+|                | `*`                        |           5 |             150 |
+
+### Growth Plan Rate Limits
+
+| Environment    | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------- | :------------------------- | ----------: | --------------: |
+| **Production** | `/sessions/whoami`         |         800 |           18000 |
+|                | `/admin/oauth2/introspect` |         800 |           18000 |
+|                | `/relation-tuples/check`   |         800 |           18000 |
+|                | `GET /admin/identities`    |          20 |             600 |
+|                | `*`                        |         400 |            9000 |
+| **Staging**    | `/sessions/whoami`         |          10 |              30 |
```

Comment:
staging has currently the same rates as dev, therefore i dont think it make sense to repeat that here. Clients already did assume that staging has more rates than developer in the past, so this need to be clarified better